app/assets/javascripts/projects/index.js

```diff
@@ -519,7 +519,11 @@ var ProjectsIndex = (function() {
 
     $(projectsPageSelector)
       .on('ajax:success', '.change-projects-view-type-form', function(ev, data) {
-        $(cardsWrapper).removeClass('list').addClass(data.cards_view_type_class);
+        if (data.cards_view_type_class === 'list') {
+          $(cardsWrapper).removeClass('cards').addClass(data.cards_view_type_class);
+        } else if (data.cards_view_type_class === 'cards') {
+          $(cardsWrapper).removeClass('list').addClass(data.cards_view_type_class);
+        }
```

Comment:
Can be written shorter as:
`$(cardsWrapper).removeClass('list cards').addClass(data.cards_view_type_class);`